k8s/base/ingress.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
This `ingress` resource is only used in the phase of issuing a certificate. If we were using HTTP, it would be used to route traffic based on hostname and paths. But right now, the `nginx` is doing the proxing directly to the PODs.